### ISSUE TITLE
Skipping entity with Id Spam as soon as Player left RFTools dimension.

### ISSUE THREAD

**Issue by Moustik2002** [McJtyMods/RFTools#372]
Hi,

I've a strange issue which is a pain in my ass because i've no decription log to provide. 
The fact is when any player is on our main RFtools Dimension TPS is nominal, but as soon as there is no more player in it (disconnected or in other dimension like overwolrd for instance), server console is spamming "Skipping entity with ID" message and TPS is flirting with 0. Does someone encounter same issue ? I can provide any relevant log if asked.

Thx in advance for your support.

**Comment by McJty** [comment]
What kind of things do you have in that dimension? Are there chunkloaders? Any special machines?

**Comment by Moustik2002** [comment]
It is a Void dimension which is used as our main dimension where our bases are located. So all kind of machines are there and the chunkload is "powered" by Immibis Dimensional Anchor.

**Comment by McJty** [comment]
Was it always like this or did this only start recently? And perhaps something has changed that could have triggered this problem (pack update or whatever)? Trying to find something that can be a clue

**Comment by Moustik2002** [comment]
I'm actually trying to find out which mod could trigger this by deactivate them one by one. But, so far, i didn't find out anything.

**Comment by Moustik2002** [comment]
Btw McJty can you tell me where i can find some infos to reimport a RFtools Dimension in a brand new overworld? I'm pretty sure my old overworld is a bit corrupted. I would like to generate a new one and import back the dimension where are located all player homes from my SMP server.

**Comment by McJty** [comment]
Oh sorry for the late reply but didn't notice this question until now. Anyway, here is the answer: https://github.com/McJty/RFTools/wiki/RFTools-Dimension-Export

**Comment by wekateka** [comment]
Hello,

Starting last night I have also started getting this behavior. I am hosting a small dedicated server at home, and when I SSH'd in this morning to check the message "Skipping entity with ID" ocurred about 5 times per second in the logs.

Reading this old ticket I tried joining the server and going to the (RFTools) dimension I build in. The messages ceased. Once I exit the dimension the spam starts again.

The dimension is a void word, with a main floating island (built, not generated) containing a lot of blocks from a lot of different mods. No new mods were added just before the problem started happening, and no config changes that removed an existing (tile)entity were passed for quite a while. The floating island is loaded by a ChickenChunks chunkloader.

The version of RFtools was 3.20. Realizing this was ancient, I updated to 4.21. Unfortunatly, the issue persisted with the exact same behaviour as before. Cool new textures though ;)

I will update here when I discover more. If there is any info I could supply you with to help solving this, please feel free to ask. I have some hobby forge/java experience so feel free to go all technical on me ;)

Attached is a (very messy, apologies) startup log from forge. GitHub does not like .log, so .log.txt it is...
[latest.log.txt](https://github.com/McJty/RFTools/files/111200/latest.log.txt)

**Comment by XFactHD** [comment]
Can you reproduce this if you replace the chickenchunks chunkloader with a chunkloader from MFR and setting it up so that the chunk with the curative vat is not loaded?

**Comment by wekateka** [comment]
Just now I tried removing the spotloader in the chunk with the vat, removed my chickenchunks loader and placed a MFR loader with the same radius (and powered it, of course). After using /chunkloaders to confirm it indeed loading the same chunks as my chickenchunks one did, I left the dimension. The spam did not appear. Same for leaving the game completely.

However, removing the MFR one and placing the CC one back without the solving spotloader, also did not bring back the spam. I tried multiple setups that gave the spam before, but unfortunately (or for anything but this research, fortunately) the spam did not return.

tl;dr Could not reproduce with MFR, but original "faulty" ChickenChunks setup also did not reproduce anymore...

If this problem returns in any way I will update here.
